content/docs/v3.4.0/faq.md

```diff
@@ -152,9 +152,9 @@ etcd sends a snapshot of its complete key-value store to refresh slow followers
 [new_issue]: https://github.com/etcd-io/etcd/issues/new
 [backend_commit_metrics]: ./metrics.md#disk
 [raft]: https://raft.github.io/raft.pdf
-[backup]: https://github.com/etcd-io/etcd/blob/master/Documentation/op-guide/recovery.md#snapshotting-the-keyspace
+[backup]: https://github.com/etcd-io/website/blob/master/content/docs/v3.4.0/op-guide/recovery.md#snapshotting-the-keyspace
```

Comment:
We can probably link directly to the page on site rather than linking to the repo here
```suggestion
[backup]: ./op-guide/recovery#snapshotting-the-keyspace
```